tools/screenshot-test/functions/data_image.ts

```diff
@@ -0,0 +1,39 @@
+import * as firebaseFunctions from 'firebase-functions';
```

Comment:
Can you use dashes in the filenames instead of underscores?

I would also call this file `write-images.ts`